gensrc/thrift/Types.thrift

```diff
@@ -129,6 +129,7 @@ enum TAggregationType {
     MIN,
     REPLACE,
     HLL_UNION,
+    BITMAP_COUNT,
```

Comment:
I'm afraid that changing NONE's ordinal will lead FE/BE meta corruption

Reply:
OK. I will keep the NONE's ordinal.